CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Upcoming Release
 
+- Added dev containers support, by [@2-5](https://github.com/2-5) in [PR 4417](https://github.com/gradio-app/gradio/pull/4417).
```

Comment:
This PR does not need a changelog entry this the change is not tied to any specific version of gradio

```suggestion
```